package-res/reportviewer/reportviewer.js

```diff
@@ -639,35 +637,19 @@ define([ 'common-ui/util/util', 'common-ui/util/timeutil', 'common-ui/util/forma
         this.view.resize();
       },
 
-      submitReportStart: function() {
-        // Submit button was mouse-downed!
-
-        // In case that a focusout already issued a "fetchParameterDefinition", and a response is to be received,
-        // we upgrade that request to behave like if a button had been clicked, by setting prompt.mode to 'MANUAL'.
-        // This way, if the user takes longer to release the mouse button,
-        // than the focusout response takes to replace the being clicked button by another one (Dashboards.init recreates everything),
-        // causing the click event no be generated,
-        // the focusout response processing will behave as if it were the response of a click.
-        //
-        // Because Dashboards.processChange ends up calling postChange in a setTimeout...
-        // this call actually gets executed **before** the logic fired by the focusout path,
-        // and fetchParameterDefinition has not been called yet.
-        this.reportPrompt.clicking = true;
+      beforeUpdateCallback: function() {
+        this.reportPrompt._isUpdatingPrompting = true;
+      },
+
+      afterUpdateCallback: function() {
+        this.view.updateLayout();
+        this._updateReportContent();
```

Comment:
do you need to add "this.reportPrompt._isUpdatingPrompting = false;" in this call?